gramps/gui/configure.py

```diff
@@ -1324,6 +1324,15 @@ def add_data_panel(self, configdialog):
         grid.attach(lwidget, 1, row, 1, 1)
         grid.attach(obox, 2, row, 2, 1)
 
+        row += 1
+        # Birthday on february 29
```

Comment:
Do we need a third option of not displaying the birthday?

Reply:
> Do we need a third option of not displaying the birthday?

This is only to allow the user to display the birthday on February 28 or March 1 for non leap years

Reply:
Perhaps some people would prefer the existing functionality of only displaying the birthday every 4 years.  It does seem unlikely though.

Reply:
[Like this guy?](https://www.youtube.com/watch?v=W7lEunJvRUU)

Reply:
Instead of a binary choice, I could set a 0228, 0301 and 0229 choice

Reply:
This is not only for birth but also for death and marriage in calendars.